src/Data/Audience.elm

```diff
@@ -34,6 +43,63 @@ type alias Audience =
     , folder : Maybe Int
     }
 
+audienceTypeToString : AudienceType -> String
+audienceTypeToString t =
+    case t of
+        Shared ->
+            "Shared"
+        Curated ->
+            "Curated"
+        Authored ->
+            "Authored"
+
+audienceTypeToIcon : AudienceType -> Html.Html msg
+audienceTypeToIcon audienceType =
+    case audienceType of
+        Authored ->
+            Icons.person
+        Curated ->
+            Icons.curated
+        Shared ->
+            Icons.people
+
+
+audienceTypeDecoder : JD.Decoder AudienceType
+audienceTypeDecoder =
+    let
+        typeToAudienceType t =
+            if t == "curated" then
+                JD.succeed Curated
+            else if t == "shared" then
+                JD.succeed Shared
+            else if t == "user" then
+                JD.succeed Authored
+            else
+                JD.fail <| """valid `type` strings are either "curated", "shared", or "user". Instead, we got: """ ++ t
```

Comment:
Why if/else here? With this amount of branches it's `case` probably better.

Reply:
I usually use if/else with strings because for some reason it got stuck in my head from the very beginning that "oh no, these are strings, so you're going to have to add `case` expressions for every possible string ever." Even though you can obviously just add a "_" at the bottom to cover any other case. But I will consider changing my ways. Having a case expression is probably a bit less noisy.

Reply:
I believe it all compiles down to the same thing; I guess we prefer `case string of ...` because of how it looks "tabular"?
Anyways I'd consider it idiomatic, but it's not a big deal at all.

Reply:
I was just curious about your preferences and I agree, `case` is less noisy and by that more obvious for the first quick look 👍